targets/unity-v2/source/Shared/Public/PluginManager.cs

```diff
@@ -100,32 +100,45 @@ private ITransportPlugin CreatePlayFabTransportPlugin()
                 transport = new PlayFabWebRequest();
 #endif
 
-#if !UNITY_2018_2_OR_NEWER // PlayFabWww will throw warnings as Unity has deprecated Www
+#if UNITY_2018_2_OR_NEWER // PlayFabWww will throw warnings as Unity has deprecated Www
+            if (transport == null)
+                transport = new PlayFabUnityHttp();
+#elif UNITY_2017_2_OR_NEWER
             if (PlayFabSettings.RequestType == WebRequestType.UnityWww)
-            {
                 transport = new PlayFabWww();
-            }
-#endif
-
+            
             if (transport == null)
                 transport = new PlayFabUnityHttp();
+#else
+            if (transport == null)
+                transport = new PlayFabWww();
+#endif
```

Comment:
This looks like a bad merge.
Unless you're rolling it back on purpose.

Reply:
Victor was mentioning there were some issues with other versions of Unity where the Www class should probably still work

Reply:
Me and Paul just discuss this offline, and I explained why this is a necessary fix